webapp/services/get_all_orders.py

```diff
@@ -36,7 +36,7 @@ def get(self):
 					beer_name = "Unknown beer id " + beer_id
 				
 				order_beer_list.append({
-					'beer_id' : order_beer.beer_id,
+					'id' : order_beer.beer_id,
```

Comment:
I think I'll need to make changes to the layout of the webapp before I can deploy this.

Reply:
The changes aren't needed for the layout as the ID of the beer isn't printed, just the name, but this single `id` naming confuses me. There's three ids in an OrderBeer (OrderBeer.id, Beer.id, Order.id) this kinda makes it look like this is the ID of the OrderBeer.

Reply:
However we decide, but we have to keep it consistent. Otherwise we cannot reuse Android models since those are automagically serialized. Therefore fields must be the same.

Reply:
Is this for the BeerAndQuantity model? I'm ok with either, let's just go with 'id' since it means no refactoring anything.